tests/sentry/api/endpoints/test_team_time_to_resolution.py

```diff
@@ -0,0 +1,108 @@
+from datetime import timedelta
+
+from django.utils.timezone import now
+
+from sentry.models import GroupHistory, GroupHistoryStatus
+from sentry.testutils import APITestCase
+from sentry.testutils.helpers.datetime import before_now
+
+
+class TeamTimeToResolutionTest(APITestCase):
```

Comment:
Let's use `freeze_time` here. I have a feeling this test will be unstable if it runs across day boundaries

Reply:
You can add 
`endpoint = "sentry-api-0-team-time-to-resolution"` to this class, then when making requests you can use

`response = self.get_success_response(self.team.organization.slug, self.team.slug)`

This automatically checks the 200 for you and builds the url, just makes tests a little DRYer